src/test/java/de/tudarmstadt/ukp/wikipedia/parser/ModularParserTest.java

```diff
@@ -0,0 +1,76 @@
+package de.tudarmstadt.ukp.wikipedia.parser;
+
+import de.tudarmstadt.ukp.wikipedia.parser.mediawiki.ModularParser;
+import org.apache.commons.math3.util.Pair;
+import org.junit.Test;
+
+
+public class ModularParserTest {
+	@Test
+	public  void testExtractNETopic(){
+		Pair<String, String> noNE = ModularParser.extractNETopic("::::::::SansNE");
+		assert(noNE.getFirst()==null);
+		assert(noNE.getSecond().equals("SansNE"));
+
+		Pair<String, String> noNE1 = ModularParser.extractNETopic("SansNE");
+		assert(noNE1.getFirst()==null);
+		assert(noNE1.getSecond().equals("SansNE"));
+
+		Pair<String, String> withNE = ModularParser.extractNETopic("fr:SansNE");
+		assert(withNE.getFirst().equals("fr"));
+		assert(withNE.getSecond().equals("SansNE"));
+
+		Pair<String, String> withNE1 = ModularParser.extractNETopic(":fr:SansNE");
+		assert(withNE1.getFirst().equals("fr"));
+		assert(withNE1.getSecond().equals("SansNE"));
+
+		Pair<String, String> withNE2 = ModularParser.extractNETopic(":::fr:SansNE");
+		assert(withNE2.getFirst().equals("fr"));
+		assert(withNE2.getSecond().equals("SansNE"));
+
+		Pair<String, String> withNE3 = ModularParser.extractNETopic(":::fr:h20:japanese");
+		assert(withNE3.getFirst().equals("fr"));
+		assert(withNE3.getSecond().equals("h20:japanese"));
+
+		Pair<String, String> withNE4 = ModularParser.extractNETopic("h20:japanese");
+		assert(withNE4.getFirst().equals("h20"));
```

Comment:
Is that really how it's supposed to be broken down? I thought it would be `("", "h20:japanese")`